pages/_app.tsx

```diff
@@ -1,30 +1,13 @@
 import { ThemeProvider } from 'emotion-theming'
-import App, { Container } from 'next/app'
 import React from 'react'
 import { theme } from '../components/utils/styles/theme'
 
-class CustomApp extends App {
-  static async getInitialProps({ Component, ctx }) {
-    let pageProps = {}
-
-    if (Component.getInitialProps) {
-      pageProps = await Component.getInitialProps(ctx)
```

Comment:
I assume this is now just part of the default next App base class?

Reply:
They removed it. You only need that with blocking script loading. All we were doing is adding the theme provider but you had to include that whereas now you don't.